clients/da-vinci-client/src/main/java/com/linkedin/davinci/storage/chunking/ChunkingUtils.java

```diff
@@ -88,9 +88,68 @@ static <VALUE, ASSEMBLED_VALUE_CONTAINER> VALUE getFromStorage(
         null,
         null,
         null,
+        false,
         false);
   }
 
+  static <VALUE, ASSEMBLED_VALUE_CONTAINER> VALUE getReplicationMetadataFromStorage(
+      ChunkingAdapter<ASSEMBLED_VALUE_CONTAINER, VALUE> adapter,
+      AbstractStorageEngine store,
+      int partition,
+      ByteBuffer keyBuffer,
+      ReadResponse response) {
+    return getFromStorage(
```

Comment:
At some points, we need to refactor these functions to use some param builder since we are passing a lot of params and the meanings are not very clear when passing a lot of `null` and booleans.

Reply:
Sure, I can cut another improvement ticket to track this issue.